MMM-MBTA.js

```diff
@@ -554,7 +581,7 @@ Module.register("MMM-MBTA", {
             for (let pred = 0; pred < data["data"].length; pred++) {
                 routeId = data.data[pred].relationships.route.data.id;
                 tripId = data.data[pred].relationships.trip.data.id;
-                alertIds = data.data[pred].relationships.alerts.data;
+                alertIds = data.data[pred].relationships.alerts ? data.data[pred].relationships.alerts.data : [];
```

Comment:
Does nullish coalsescing work? I think it's been long enough that it should be supported at this point
```suggestion
                alertIds = data.data[pred].relationships.alerts?.data ?? [];
```

Reply:
Much cleaner, thanks!